src/libraries/BoringERC20.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+// With thanks to @BoringCrypto
+pragma solidity ^0.8.0;
+import "../interfaces/IERC20.sol";
```

Comment:
We can use the IERC20 interface in forge-std. This way the local one can be removed.

`import "forge-std/interfaces/IERC20.sol";`